hugo/content/privacy.html

```diff
@@ -5,98 +5,98 @@
 type: oct
 ---
 
-<div class="text-content">
-    <h1>Privacy Policy</h1>
+<div class="w-4/5 max-w-[850px] mt-[50px] font-urbanist leading-[1.35rem]">
+    <h1 class="text-[2.25rem] leading-[2.5rem] my-[0.83em] font-bold">Privacy Policy</h1>
     <p><strong>10 July 2024</strong></p>
-    <p>This Privacy Policy describes how TypeFox GmbH (“Company”, “we”, “us”, or “our”) collects, uses, and shares
-        personal data when you use the Open Collaboration Tools service on this website (the “Service”). We are
+    <p class="my-4">This Privacy Policy describes how TypeFox GmbH ("Company", "we", "us", or "our") collects, uses, and shares
+        personal data when you use the Open Collaboration Tools service on this website (the "Service"). We are
         committed to protecting your privacy and handling your personal data in a transparent and secure manner.</p>
-    <h2 id="1-data-controller">
+    <h2 id="1-data-controller" class="text-[1.5rem] leading-[1.75rem] font-bold text-center my-[0.83em]">
```

Comment:
I think it was not intended to have these headings centered. I suggest changing them to normal (same for tos.html)

Reply:
Done